android/java/org/chromium/chrome/browser/BraveRewardsUserWalletActivity.java

```diff
@@ -124,6 +125,11 @@ private void SetDisconnectBtnClickHandler() {
         */
     }
 
+    @Override
+    public boolean shouldStartGpuProcess() {
+        return true;

```

Comment:
why do we need that?

Reply:
The method is abstract. It needs to be implemented in every subclass.

Reply:
@SergeyZhukovsky I think it's an abstract method. So we need to override ir

Reply:
in what cases we return true and false? As I see false in that place https://github.com/brave/brave-core/pull/9256/files#diff-cc8f95891980e64d85e01f6b430c97a504ab9f158e6f20c0e1d1e9a55583809bR75

Reply:
My understanding is if the activity doesn't need to present a GUI then it should be false, as in `SharedClipboardShareActivity.java` and `TrustedVaultKeyRetrievalProxyActivity.java`.